### ISSUE TITLE
wrong echo syntaxe in the construction of the https check

### ISSUE THREAD

**Issue by manchot0** [matteocorti/check_ssl_cert#115]
the -e parameter of the echo on line 467 of the script is actually printed in the url, not interpreted.

```
exec_with_timeout "$TIMEOUT" "echo -e '${HTTP_REQUEST}' | $OPENSSL s_client ${CLIENT} ${CLIENTPASS} -crlf -ign_eof -connect $HOST:$PORT ${SERVERNAME} -showcerts -verify 6 ${ROOT_CA} ${SSL_VERSION} ${SSL_VERSION_DISABLED} ${SSL_AU} 2> ${ERROR} 1> ${CERT}"
```

Here is what i got in my nginx log : 

> [16/Jan/2019:14:02:44 +0100] 192.168.32.205 - - - 35926 "-e HEAD / HTTP/1.1" 400 - 150 "-" "-" "TLSv1.2" "ECDHE-RSA-AES256-GCM-SHA384" - [-] [-]

we can see the -e being printed before the method HEAD which break my log parser. 

I removed the -e and it solved the problem for me but i don't know if its really safe to suppress it, i guess he is not interpreted so it should be fine ?

It may be necessary to do the same for the smtp and irc protocols.

**Comment by matteocorti** [comment]
-e was introduced by a patch an I did not notice that it is not POSIX compliant. I'll look for a solution

**Comment by matteocorti** [comment]
Fixed with 1.80.1

**Comment by mobitux** [comment]
For HTTPS this was not implemented correctly in commit 73b81b5.
Instead of replacing "echo -e" with "printf" only the "-e" was removed.
